### PR TITLE
rebase 3.0.0 beta2+quic

### PR DESCRIPTION
**Pull request by tmshort**
Added `QUIC:` to some of the headlines. Otherwise, this was a relatively clean rebase.

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by nibanks — APPROVED** [review]

**Review by kaduk — APPROVED** [review]

**Comment by tmshort** [comment]
Waiting until checks pass....

File: README.md [hunk 1]
```diff
@@ -1,223 +1,113 @@
...
+- The changes for QUIC will always be at the tip of the branch -- you will know what
+  is from the original OpenSSL and what is for QUIC.
+- New versions are quickly created once upstream creates a new tag.
+- The use of git commands (such as "cherry") can be used to ensure that all changes
 
-If you intend to contribute to OpenSSL, either to fix bugs or contribute
-new features, you need to fork the OpenSSL repository openssl/openssl on
-GitHub and clone your public fork instead.
+have moved forward with minimal or no changes. You will be able to see
+QUIC: Add X" on all branches and the commit itself will be nearly identical on all
```
Comment (kaduk) [inline README.md:61]: (We should probably add the missing double-quote mark at the start of the line in a follow-up)

File: README.md [hunk 2]
```diff
@@ -1,223 +1,113 @@
...
-GitHub pull requests. For more details, see [Contributing](#contributing).
+Library names will be the same, but will use a different version number. The version
+numbers for the current OpenSSL libraries are `1.1` (for the 1.1.0 and 1.1.1 branches)
+and `3` (for the to-be-3.0 branch). We will be prefixing 81 (ASCII for 'Q') to
+the version numbers to generate a unique version number.
 
-Build and Install
-=================
+- libcrypto.so.81.3 vs libcrypto.so.3
+- libcrypto.so.81.1.1 vs libcrypto.so.1.1   libcrypto.so.3
```
Comment (kaduk) [inline README.md:73]: (likewise the stray "libcrypto.so.3" could be removed in a follow-up)

File: README.md (hunk 1 above)
Comment (tmshort) [reply README.md]: Fixed in tip commit.

File: README.md (hunk 2 above)
Comment (tmshort) [reply README.md]: Fixed in tip commit

**Review by kaduk — APPROVED** [review]
Fixups look good, thanks

**Review by richsalz — APPROVED** [review]
nice job.

**Comment by tmshort** [comment]
"CIFuzz/Fuzzing" has always failed

**Pull request merged**